index.js

```diff
@@ -223,9 +223,9 @@ module.exports.spawnPythonService = (additionalEnv = {}) => {
 function findBestSubtitleFile(list, locales = []) {
   // favor locales but some subtitles just have the language code, zip it to keep the ordering
   const localesAndLanguages = locales.map((locale) => [locale.toLowerCase(), locale.substring(0, 2)]).flat();
+  const languages = [...localesAndLanguages, ...EN_LIST];
   // unfound languages will have a priority of -1, so reversing the list here
-  const languages = [...localesAndLanguages, ...EN_LIST].reverse();
-  const uniqueLanguages = languages.filter((x, index) => languages.indexOf(x) === index);
+  const uniqueLanguages = languages.filter((x, index) => languages.indexOf(x) === index).reverse();
```

Comment:
The reason that when we put in ['en-US', 'de-DE'] and it returns the subtitle in "de" is because 'en-US' is provided as fallback in EN_LIST and they get removed in the unique call.
For example, languages is [ 'en-au', 'en-gb', 'en', 'en-us', 'de', 'de-de', 'en', 'en-us' ] after merging and the unique call will get rid of the `en` and `en-us` at the end so that will not have the highest priority anymore.
The trick is just to do the reverse after we do the unique call.